pkg/apis/metal3/v1alpha1/baremetalhost_types.go

```diff
@@ -157,6 +526,9 @@ type BareMetalHostSpec struct {
 	// How do we connect to the BMC?
 	BMC BMCDetails `json:"bmc,omitempty"`
 
+	//BIOS configuration for bare metal server
+	BIOS *BIOSConfig `json:"bios,omitempty"`
```

Comment:
This should be "Firmware" https://github.com/metal3-io/metal3-docs/pull/63

Reply:
Thanks, done.

Reply:
Sorry for the late reply. Forgot to subscribe to this PR.